app/views/books/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1> Title: <%= @book.title %> </h1>
+
+<h2> Average Rating: <%= @book.average_rating %>
+<h2> Highest Rating: <%= @book.highest_rating.rating %>
+  <h3> Reviewer: <%= @book.highest_rating.user.name %> </h3>
```

Comment:
This is a little too much chaining in the view, I would think a little more about what i can pass in to have all this information available while not having to iterate through 3 objects and their attributes.